examples/pipelines/datacomp/components/download_images/src/main.py

```diff
@@ -0,0 +1,159 @@
+"""
```

Comment:
What are the reasons for duplicating this component and not making the changes directly to the one in the registry?